examples/model_compress/sparse_learning_example/main_tf_sparse_pruner.py

```diff
@@ -0,0 +1,114 @@
+import tensorflow as tf
```

Comment:
suggest to use one folder for pytorch version and one folder for tensorflow version